tpp/src/main/Representable.js

```diff
@@ -1,5 +1,5 @@
 // @flow
```

Comment:
You should bump tpp version afaik

Reply:
oh ya, the version was bumped again. I will bump it to 1.0.12